.changeset/config.json

```diff
@@ -12,5 +12,7 @@
   "access": "public",
   "baseBranch": "develop",
   "updateInternalDependencies": "patch",
-  "ignore": []
+  "ignore": [
+    "ledger-live-desktop"
```

Comment:
Not sure about this change :thinking: should I remove it?

Reply:
Yep absolutely!